apps/web/ee/lib/workflows/reminders/smsReminderManager.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  WorkflowTriggerEvents,
+  TimeUnit,
+  WorkflowTemplates,
+  WorkflowActions,
+  WorkflowMethods,
+} from "@prisma/client/";
+import dayjs from "dayjs";
+import isBetween from "dayjs/plugin/isBetween";
```

Comment:
```suggestion
import dayjs from "@calcom/dayjs";
```
Let's import the shared dayjs package instead since it already has all needed plugins so we can have a single source of truth.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix